src/Middleware/WebSockets/src/WebSocketOptions.cs

```diff
@@ -28,6 +27,7 @@ public WebSocketOptions()
         /// Gets or sets the size of the protocol buffer used to receive and parse frames.
         /// The default is 4kb.
         /// </summary>
+        [Obsolete("This is obsolete and will be removed in a future version")]
```

Comment:
Assuming the property doesn’t currently do anything, perhaps: `Setting this property has no effect. It will be removed in a future version.`